pico/runtime/src/main/java/io/helidon/pico/runtime/Invocation.java

```diff
@@ -118,18 +118,18 @@ public static <T> List<Provider<T>> mergeAndCollapse(List<Provider<T>>... lists)
 
     @Override
     public V proceed(Object... args) {
-        if (!interceptorIterator.hasNext()) {
+        if (interceptorIterator.hasNext()) {
```

Comment:
it is easier to reason about an if statement if negation is not used (and I had to debug this method a lot).
This is a best practice we use in Helidon, fixed this instance as PR is related.